pkg/manager/diff.go

```diff
@@ -818,23 +818,18 @@ func affectedFiles(cfg *mgrconfig.Config, gitPatches [][]byte) (direct, transiti
 		if strings.HasSuffix(file, ".h") && cfg.KernelSrc != "" {
 			// Ideally, we should combine this with the recompilation process - then we know
```

Comment:
Can you please add a comment here explaining that we're scanning the .c files in KernelSrc for <file.h>?

Reply:
Added a comment.